daemon/logger/splunk/splunk.go

```diff
@@ -63,6 +64,8 @@ const (
 	envVarStreamChannelSize     = "SPLUNK_LOGGING_DRIVER_CHANNEL_SIZE"
 )
 
+var batchSendTimeout = 30 * time.Second
```

Comment:
Why not const?

Reply:
I'm changing it to a shorter timeout for tests.

Reply:
:+1: